ide/vagrant/Vagrantfile

```diff
@@ -15,14 +18,14 @@ sudo /xcsoar-host-src/ide/provisioning/install-debian-packages.sh
 sudo apt-get clean
 
 cd
-git clone -o host /xcsoar-host-src xcsoar-src
+git clone --recurse-submodules -o host /xcsoar-host-src xcsoar-src
```

Comment:
The git checkout on the host system should already be a full checkout with submodules, so why should we git-clone it in the provisioning script instead of using it directly?
Since the host checkout is cloned only once at provisioning time, it might get outdated when the devloper modified it after the first provisioning of the box.

Reply:
Even if the submodules are checked out locally, git clone alone will not clone the submodules. What bothers me is that it actually checks them out from the defined upstream sources, and doesn't copy it from the host source tree. So a change in a submodule could be lost, or the state could differ from other reason. 

Options that we could have a look at:
- use rsync instead of git clone
- use host export mount

For the moment this works, so it can be applied.